perllib/FixMyStreet/App/Controller/Admin/Reports.pm

```diff
@@ -234,6 +234,18 @@ sub edit : Path('/admin/report_edit') : Args(1) {
             push @fields, { name => 'Duplicates', val => join( ',', @{ $problem->get_extra_metadata('duplicates') } ) };
             delete $extra->{duplicates};
         }
+
+        if ( $extra->{contributed_by} ) {
+            my $u = $c->cobrand->users->find({id => $extra->{contributed_by}});
```

Comment:
If `$u` isn't found then this causes a crash on the next line. This is unlikely to happen, but is possible if e.g. a superuser contributes a report and then tries to view the report admin page on the cobrand site. Because `UKCouncils::users_restriction` doesn't include superusers it won't be found.